### ISSUE TITLE
Triplet Loss or Softmax

### ISSUE THREAD

**Issue by chengyawlow** [timesler/facenet-pytorch#72]
May I know if the embedding part is trained using triplet loss? or softmax cross-entropy instead? seeing that the pre-trained model includes also a softmax classifier. thank you~

**Comment by timesler** [comment]
The embedding model is train with cross-entropy, as shown [here](https://github.com/davidsandberg/facenet/wiki/Training-using-the-VGGFace2-dataset). The model was not trained with code in this repo, but in the tensorflow facenet repo.
